portal-web/test/functional/com/liferay/portalweb/paths/pathlib/uisections/CKEditor.path

```diff
@@ -153,6 +158,17 @@
 	<td>//div[label[contains(.,'Video')]]//input</td>
 	<td></td>
 </tr>
+<!--SOURCE_CODE_PREVIEW-->
+<tr>
+	<td>CODE_PREVIEW_IFRAME</td>
+	<td>//*[contains(@class,'code_preview')]//iframe</td>
+	<td></td>
+</tr>
+<tr>
+	<td>SOURCE_CODE_DIALOG</td>
+	<td>//*[contains(@class,'cke_dialog_contents')]//*[contains(@class,'CodeMirror')]</td>
```

Comment:
I'm not sure what this path is supposed to point to exactly since there's multiple possible elements.

![screenshot_25](https://user-images.githubusercontent.com/35235266/93118073-91bddd80-f674-11ea-805a-45514ca215f8.png)